### DISCUSSION TITLE
Accessabilty/PUSH

### DISCUSSION THREAD

**Discussion by anish-Sherghati** [community/community#135927] [Programming Help]
### Select Topic Area

General

### GitHub Feature Area

GitHub Education

### Body

Hi Team, 
Currently working in Gitlab remote Account A  and also login in my local Git Bash with Account A.
Doing everything daily push or pull with this account, working nine repository.

Now I wanted to Push all nine repository to Gitlab Account B. How can I do any suggestion most welcome.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @anish-Sherghati ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
